templates/zerver/help/message-formatting.md

```diff
@@ -46,6 +46,12 @@ To include hyperlinks in your messages, you can either enter the link's url addr
 
 Zulip currently does not support image embedding through links.
 
+## Stream Links
+
+To link to another stream in one of your messages, you can either start with `#` and surround your message with double asteriks `**`, or you can type `#streamName` and click the stream name in the popup.
```

Comment:
Can you swap the order of these options? I think we want the easy one to be first.